lib/sisimai/bite/json.rb

```diff
@@ -0,0 +1,38 @@
+module Sisimai
+  module Bite
+    module JSON
+      # Sisimai::Bite::JSON - Base class for Sisimai::Bite::JSON::*
+      class << self
+        # Imported from p5-Sisimail/lib/Sisimai/Bite/JSON.pm
+        require 'sisimai/bite'
+        require 'sisimai/rfc5322'
+
+        def headerlist; return []; end
+        def pattern;    return []; end
+
+        # MTA list
+        # @return   [Array] MTA list with order
+        def index
+          return %w|SendGrid AmazonSES|
+        end
+
+        # Convert from JSON object to Sisimai::Message
+        # @param         [Hash] mhead       Message header of a bounce email
+        # @param         [String] mbody     Message body of a bounce email(JSON)
+        # @return        [Hash, Undef]      Bounce data list and message/rfc822 part
+        #                                   or Undef if it failed to parse or the
+        #                                   arguments are missing
+        def scan; return nil; end
+
+        # @abstract      Adapt bounce object for Sisimai::Message format
+        # @param         [Hash] argvs       bounce object returned from each email cloud
+        # @return        [Hash, Undef]      Bounce data list and message/rfc822 part
+        #                                   or Undef if it failed to parse or the
+        #                                   arguments are missing
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->